Source/EasyNetQ/Extentions.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+
+namespace EasyNetQ
+{
+    public static class Extentions
+    {
+        public static void SafeDispose(this IDisposable disposable)
```

Comment:
this should be always the expected behavior of all Dispose methods, does IModel.Dispose throw exception? If yes, we should notify to the RabbitMQ.Client team